flink-tests/src/test/java/org/apache/flink/test/checkpointing/AutoRescalingITCase.java

```diff
@@ -337,7 +337,7 @@ public void testCheckpointRescalingNonPartitionedStateCausesException() throws E
 
             restClusterClient.updateJobResourceRequirements(jobID, builder.build()).join();
 
-            waitForRunningTasks(restClusterClient, jobID, parallelism2);
+            waitForRunningTasks(restClusterClient, jobID, 2 * parallelism2);
```

Comment:
this is out-of-scope for FLINK-34336? Was it accidentally added?

Reply:
The last line calls the rescale api, IIUC, `waitForRunningTasks` should wait for all tasks of rescaled job. 
The task number after rescaling should be `2 * parallelism2`.

Other tests are similar with here, but other tests wait for all task of rescaled job.